backend/open_webui/retrieval/loaders/external_document.py

```
@@ -1,5 +1,5 @@
 import requests
-import logging
+import logging, os
```

Ruff (0.11.9)

Multiple imports on one line

Split imports

(E401)